### PR TITLE
Dx 116

### PR DESCRIPTION
**Pull request by souravg-db**

### CODE REVIEW

File: discoverx/scanner.py [hunk 1]
```diff
@@ -133,7 +134,15 @@ def save(self, scan_table_name: str):
 
         logger.friendly(f"Overwrite scan result table {scan_table_name}")
 
-        scan_result_df.write.format("delta").mode("overwrite").saveAsTable(scan_table_name)
+        # scan_result_df.write.format("delta").mode("overwrite").saveAsTable(scan_table_name)
```
Comment (edurdevic) [inline discoverx/scanner.py:137]: We can remove this comment I think

File: discoverx/scanner.py [hunk 2]
```diff
@@ -133,7 +134,15 @@ def save(self, scan_table_name: str):
...
+        scan_delta_table.alias("scan_delta_table").merge(
+            scan_result_df, "scan_delta_table.table_catalog = scan_result_df.table_catalog \
+            and scan_delta_table.table_schema = scan_result_df.table_schema \
+            and scan_delta_table.table_name = scan_result_df.table_name \
+            and scan_delta_table.column_name = scan_result_df.column_name ") \
```
Comment (edurdevic) [inline discoverx/scanner.py:142]: This looks correct, but it would require a unit test.

**Review by edurdevic — COMMENTED** [review]
The regex fix looks good, but the merge would be great if integrated with a unit test.

File: discoverx/scanner.py (hunk 1 above)
Comment (souravg-db) [reply discoverx/scanner.py]: updated save method test case

**Pull request closed without merging**